### DISCUSSION TITLE
Please make it easy to copy the path to a file

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#41723] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Consider this location:
https://github.com/dart-lang/sdk/blame/24e4e16ef1ec4bf38dd8d01e4af852ee70a837d5/pkg/analyzer/test/generated/patterns_parser_test.dart#L8211

You're 8,000 lines down. You can't click outside the scrolling box to focus outside the scrolling box to scroll up the page. You don't want to slowly scroll up 8,000 lines. Maybe you're on a mobile device that doesn't give easy access to a `home` key to jump to the top of the page. And you've certainly forgotten that the top of the page doesn't have a helpful path item anyway.

There is a copy button near the path that's visible, but it copies the entire contents of the file (which might kill your device, but hey).

If this was a PR and you were in the files view, there'd be a copy button next to the file path that would let you copy the file path.

Yes, copy file path is technically in `more file options`, but it's hardly discoverable and it's in the wrong place. It should be right next to the file path. Arguably a bunch of those items should probably be one row up from where they are....

**Comment by gdvalderrama** [comment]
Adding some visuals here, since this has been bothering me too.
Basically, I'd expect the copy path to also be available after scrolling.

At the top, we have the copy path icon after the path

![github_copy_path](https://github.com/community/community/assets/7897337/f5555211-e9f9-442c-ba96-65f44cfe2877)

After scrolling we no longer have this option

![github_path_scrolled](https://github.com/community/community/assets/7897337/398e7512-642d-4a41-a21f-31290a5d2980)
